experimental/logging/testdata/listener.wat

```diff
@@ -25,4 +16,9 @@
     i32.const 3 call_indirect (type 0) ;; element 3, func type 0
     drop ;; errno
   )
+
+  (func $rand (export "rand") (param $len i32)
+    local.get 0 ;; buf_len
+    call $wasi_rand
```

Comment:
this is the guest call which should be skipped when host only mode, though the fact `wasi.random_get` was initiated by the exported `rand` should show in both cases.